CONTRIBUTING.md

```diff
@@ -0,0 +1,50 @@
+# CONTRIBUTING
+
+We're excited that you're considering contributing to Zorro! This is a new project and things are moving fast. As a result, we may not have bandwidth to do as good a job onboarding new contributors or quickly reviewing pull requests as we would like. (That will eventually change as the protocol matures.)
+
+With that disclaimer in mind, if you're a self-starter and care deeply about Zorro's mission, your contributions could have a huge positive impact at this early stage! This document should help you get started.
+
+Our website is found at [https://zorro.xyz/](https://zorro.xyz/), and you can join our community on [Discord](https://discord.gg/Caj283PtN4). To learn more about our mission and goals, check out our [whitepaper](https://hackmd.io/@zorro-project/zorro-whitepaper).
+
+# Contributing Code
+
+## Project Setup
+
+### Accounts
+
+1. MetaMask
+
+    [MetaMask](https://metamask.io/index.html) is a crypto wallet & gateway to blockchain applications. You will need MetaMask to interact with the [Görli (Goerli) Test Network](https://goerli.net/), which is the test network that our application uses. Install MetaMask and set up a new account, and then change your network to the “Goerli Test Network”. You may need to change your settings to show test networks.
+
+2. Infura
+
+    [Infura](https://infura.io/) is a Web3 backend and Infrastructure-as-a-Service (IaaS) provider that offers a variety of tools for blockchain developers. You will need Infura to generate an API key to run our contracts locally. Create an account with Infura, and then create a project. You will need the Project ID for an environment variable in the project.
+
+### Setting up the repository
+
+1. Clone the repository.
+
+```jsx
```

Comment:
I must admit I'm a bit confused by your your comment https://github.com/zorro-project/zorro/pull/48#issuecomment-1013303806. What does adding `jsx` here get us?

Reply:
I wrote this in Notion, so it exported the jsx. I'll change it to the default Github code snippet syntax, but ```jsx works the same I believe -- view this https://github.com/valeseus/zorro/blob/docs/CONTRIBUTING.md

Reply:
yeah, the `jsx` tells Github to render with Javascript (with JSX) syntax highlighting. But it's a bit confusing for a reader since the provided code isn't Javascript. 😆